test/FormControlSpec.js

```diff
@@ -77,4 +77,12 @@ describe('<FormControl>', () => {
     expect(instance.input.tagName).to.equal('INPUT');
   });
 
+  it('should properly display size of FormControl', () => {
+    $(
+      <FormControl id="foo" type="text" bsSize="lg" className="bar" />
+    )
+      .render()
+      .single('input#foo.bar.form-control.input-lg');
```

Comment:
I'd simplify this selector to just `input.input-lg` so it's a bit easier to see that the important assertion bit is